packages/evm/test/precompiles/05-modexp.spec.ts

```diff
@@ -51,3 +71,81 @@ describe('Precompiles: MODEXP', () => {
     assert.strictEqual(result.executionGasUsed, gas)
   })
 })
+
+function enoughGas(opts: PrecompileInput): boolean {
```

Comment:
Now the "utility methods" are exported to calculate the gas fee here, but I think it would be more clean to instead write a gas calculator method inside the precompile and export that to calculate the required gas.